File: FlatBuffersWindows/FlatBuffersWindows/model/vo/json/CasaJSON.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FlatBuffersWindows.model.vo.json
{
    class CasaJSON : ImovelJSON
    {
        public AreaJSON areaTerreno { get; set; }
        public float areaConstruida { get; set; }
    }
}

File: FlatBuffersAndroid/app/src/main/java/com/talkitbr/flatbuffersandroid/model/vo/json/CasaJSON.java
package com.talkitbr.flatbuffersandroid.model.vo.json;

public class CasaJSON extends ImovelJSON {
    private AreaJSON areaTerreno;
    private float areaConstruida;

    public AreaJSON getAreaTerreno() {
        return areaTerreno;
    }

    public void setAreaTerreno(AreaJSON areaTerreno) {
        this.areaTerreno = areaTerreno;
    }

    public float getAreaConstruida() {
        return areaConstruida;
    }

    public void setAreaConstruida(float areaConstruida) {
        this.areaConstruida = areaConstruida;
    }
}
